.prow.yaml

```diff
@@ -1,186 +1,186 @@
+  - name: pre-dashboard-test-unit
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/dashboard.git"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/chrome-headless:v0.8
+          command:
+            - make
+            - test-headless
+          resources:
+            requests:
+              cpu: 1
+              memory: 3Gi
+            limits:
+              cpu: 2
+              memory: 5Gi
+          env:
+            - name: CODECOV_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: kubermatic-codecov
+                  key: token
 
-- name: pre-dashboard-test-e2e
-  always_run: true
-  decorate: true
-  clone_uri: "ssh://git@github.com/kubermatic/dashboard.git"
-  extra_refs:
-    # Kubermatic repo is required as we need to have access to the helm files used to deploy it.
-    - org: kubermatic
-      repo: kubermatic
-      base_ref: master
-      clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
-  labels:
-    preset-anexia: "true"
-    preset-aws: "true"
-    preset-azure: "true"
-    preset-digitalocean: "true"
-    preset-gce: "true"
-    preset-hetzner: "true"
-    preset-kubevirt: "true"
-    preset-openstack: "true"
-    preset-packet: "true"
-    preset-vsphere: "true"
-    preset-docker-pull: "true"
-    preset-goproxy: "true"
-    preset-kind-volume-mounts: "true"
-    preset-kubeconfig-ci: "true"
-  spec:
-    containers:
-      - image: quay.io/kubermatic/e2e-kind-cypress:v1.4.0
-        command:
-          - make
-          - run-e2e-ci
-        securityContext:
-          privileged: true
-        resources:
-          requests:
-            memory: 6Gi
-            cpu: 4
-          limits:
-            memory: 6Gi
-            cpu: 4
-        env:
-          - name: SERVICE_ACCOUNT_KEY
-            valueFrom:
-              secretKeyRef:
-                name: e2e-ci
-                key: serviceAccountSigningKey
+  - name: pre-dashboard-test-e2e
+    always_run: true
+    decorate: true
+    clone_uri: "ssh://git@github.com/kubermatic/dashboard.git"
+    extra_refs:
+      # Kubermatic repo is required as we need to have access to the helm files used to deploy it.
+      - org: kubermatic
+        repo: kubermatic
+        base_ref: master
+        clone_uri: "ssh://git@github.com/kubermatic/kubermatic.git"
+    labels:
+      preset-anexia: "true"
+      preset-aws: "true"
+      preset-azure: "true"
+      preset-digitalocean: "true"
+      preset-gce: "true"
+      preset-hetzner: "true"
+      preset-kubevirt: "true"
+      preset-openstack: "true"
+      preset-packet: "true"
+      preset-vsphere: "true"
+      preset-docker-pull: "true"
+      preset-goproxy: "true"
+      preset-kind-volume-mounts: "true"
+      preset-kubeconfig-ci: "true"
+      preset-minio: "true"
+    spec:
+      containers:
+        - image: quay.io/kubermatic/e2e-kind-cypress:v1.5.0
+          imagePullPolicy: Always
```

Comment:
I rebuilt the same Docker image a couple of times. To prevent some of our nodes still having a non-up-to-date image, the pull policy should be Always for a week or so.